src/Models/Like.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
```

Comment:
Microsoft docs recommends System.Text.Json(.Serialization), is there a specific reason to using the newtonsoft package? Does the default json (de)serializer not support ignoring specific fields? I'm not sure what the difference between the two is

Reply:
https://stackoverflow.com/a/65600926

Reply:
fixed in next commit